#! /bin/sh /usr/share/dpatch/dpatch-run
## htmldocs.dpatch by  <debian@adkgis.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Give correct location of html docs provided by grass-doc package

@DPATCH@
diff -urNad grass-5.7.0/scripts/g.manual/g.manual /tmp/dpep.ZJpZwa/grass-5.7.0/scripts/g.manual/g.manual
--- grass-5.7.0/scripts/g.manual/g.manual	2003-10-16 10:15:17.000000000 -0400
+++ /tmp/dpep.ZJpZwa/grass-5.7.0/scripts/g.manual/g.manual	2004-12-02 20:17:56.000000000 -0500
@@ -26,6 +26,8 @@
 #% required : no
 #%End
 
+GRASS_DOC_BASE="/usr/share/doc/grass-doc"
+
 if [ "$1" != "@ARGS_PARSED@" ] ; then
   exec "$GISBASE"/etc/bin/cmd/g.parser "$0" "$@"
 fi
@@ -39,7 +41,7 @@
 
 if [ $GIS_FLAG_i -eq 1 ] ; then
    echo "Starting browser <$BROWSERNAME> for manual page index..."
-   $GRASS_HTML_BROWSER "$GISBASE"/docs/html/index.html
+   $GRASS_HTML_BROWSER "$GRASS_DOC_BASE"/html/index.html
   exit 0
 fi
 
@@ -53,7 +55,7 @@
     exit 1
 else
     echo "Starting browser <$BROWSERNAME> for module $GIS_OPT_entry..."
-    $GRASS_HTML_BROWSER "$GISBASE"/docs/html/"$GIS_OPT_entry".html
+    $GRASS_HTML_BROWSER "$GRASS_DOC_BASE"/html/"$GIS_OPT_entry".html
 fi
 
 exit 0
